scripting/complex-script/npmAndBowerRepositories.groovy

```diff
@@ -1,15 +1,38 @@
-repository.createNpmProxy('npmjs-org', 'https://registry.npmjs.org')
+import groovy.xml.MarkupBuilder
+def newRepositories = []
+newRepositories << repository.createNpmProxy('npmjs-org2', 'https://registry.npmjs.org')
 
-repository.createNpmHosted('npm-internal')
+newRepositories << repository.createNpmHosted('npm-internal2')
```

Comment:
Why the 2? The names are the same as they are in the documentation itself on purpose...